python/lsst/ip/isr/isrStatistics.py

```diff
@@ -115,6 +117,43 @@ class IsrStatisticsTaskConfig(pexConfig.Config):
         default=[0, 5, 16, 50, 84, 95, 100],
     )
 
+    doBiasShiftStatistics = pexConfig.Field(
+        dtype=bool,
+        doc="Measure number of image shifts in overscan?",
+        default=False,
+    )
+    biasShiftFilterOrder = pexConfig.Field(
+        dtype=int,
+        doc="Filter order for highpass filter.",
```

Comment:
Is this a polynomial order or something else?

Reply:
I've clarified that this is a Butterworth highpass filter, which constrains the definition of order somewhat.